nasa-pictures-stealer/src/main/java/nasapicturesstealer/infrastructure/entity/NasaCameraEntity.java

```diff
@@ -0,0 +1,36 @@
+package nasapicturesstealer.infrastructure.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.springframework.data.annotation.CreatedDate;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
```

Comment:
Remove all args constructor and setter. Your id should be changed only by orm framework